src/server/game/Entities/Player/Player.cpp

```diff
@@ -2593,23 +2593,23 @@ void Player::GiveLevel(uint8 level)
     PlayerClassLevelInfo classInfo;
     sObjectMgr->GetPlayerClassLevelInfo(GetClass(), level, &classInfo);
 
-    // send levelup info to client
```

Comment:
Would be good to restore comments

Reply:
I would prefer to move the packet to a separate method

Reply:
Would make sense if it would be used outside of this method but it's not and also this is primarily a conversion, not a refactor